.github/workflows/release.yml

```diff
@@ -5,9 +5,7 @@ on:
     types: [published]
 
 permissions:
-  contents: write
-  issues: write
-  pull-requests: write
+  contents: read
```

Comment:
These permissions are not needed anymore - the other jobs have defined their required permissions